src/libraries/Microsoft.PowerFx.Interpreter/Marshal/TypeCoercionProvider.cs

```diff
@@ -18,7 +18,8 @@ namespace Microsoft.PowerFx
     public static class TypeCoercionProvider
     {
         internal static FormattingInfo CreateFormattingInfo() => new FormattingInfo()
-        {
+        {
+            // $$$ can't use current culture
             CultureInfo = CultureInfo.CurrentCulture,
```

Comment:
>CultureInfo.CurrentCulture [](http://example.com/codeflow?start=26&length=26)

This is recent code. 
We could probably remove this and just use the default.

Reply:
Keeping for now, not to break PA Client
Will remove when we'll remove all CurrentCulture usage